samples/wifi/radio_test/prj.conf

```diff
@@ -6,6 +6,8 @@
 CONFIG_WIFI=y
 CONFIG_WIFI_NRF700X=y
 CONFIG_NRF700X_RADIO_TEST=y
+# For BLE combo build
+CONFIG_BOARD_ENABLE_CPUNET=y
```

Comment:
maybe the docs for the sample is out-dated, but the docs also refers to nRF7002 ek as a shield.

If being used as a shield, I don't think this line is needed, is it ?

And how will things work now if building this sample for the shield combo, and not the nrf7002_nrf5340dk version.

Ref: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/wifi/radio_test/sample_description.html#building-and-running
> To build for the nRF7002 EK and nRF5340 DK, use the nrf5340dk_nrf5340_cpuapp build target with the SHIELD CMake option set to nrf7002_ek. The following is an example of the CLI command:
>
> `west build -b nrf5340dk_nrf5340_cpuapp -- -DSHIELD=nrf7002_ek`

Reply:
This line is independent of nrf7002 DK or nrf7002 shield, as this just pulls in the BLE combo image when CPUNET is enabled.

Reply:
Gentle reminder @tejlmand for review.

Reply:
right, for some reason I got the impression there was a difference between running on nRF7002dk vs nrf5340dk with nRF7002 shield.
But this pr is of course targeting the netcore on 5340 SoC regardless of setup.

Accepted.